docs/about/contributing/AI_CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Guidelines for Using Generative AI in Contributions
+
+This document defines rules for contributions where an AI system is the primary author of the code (i.e., most of the pull request was generated by AI).
```

Comment:
> most of the pull request was generated by AI)

This decision is very much "vibes based" from the reviewer's perspective. As such, I anticipate that this may be the source of a number of arguments. Unfortunately, I don't have any suggestion for a better delineation.

Reply:
I do agree, but I kept it subjective because I know we all use AI day to day in our coding. I did not want to encompass our usual PRs.

I do not like the idea of saying like "50% of lines" or something.